web/templates/base.html

```diff
@@ -63,7 +63,8 @@
 		<div id="push"></div>
 		</div>
 
-		<footer>
+		{% include 'layout/footer.html' %}
+<!--/.		<footer>
```

Comment:
is this still needed? if so, why is it commented out?
if not, why is it still here? :smile: